hail_scripts/reference_data/combine.py

```diff
@@ -132,3 +141,25 @@ def join_hts(datasets, version, reference_genome='37'):
     )
     joined_ht.describe()
     return joined_ht
+
+
+def update_existing_joined_hts(
+    destination_path: str,
+    dataset: str,
+    datasets: List[str],
+    version: str,
+    genome_version: str,
+):
+    joined_ht = hl.read_table(destination_path)
+    dataset_ht = get_ht(dataset, genome_version)
+    if 'coverage' in dataset:
+        joined_ht = (
+            joined_ht.drop(dataset)
+            .join(dataset_ht, 'outer')
+            .filter(
+                hl.any([~hl.is_missing(joined_ht[dataset]) for dataset in datasets]),
+            )
+        )
+    else:
+        joined_ht.annotate(dataset=dataset_ht[dataset_ht.locus][dataset])
```

Comment:
won't this add an annotation called `dataset`, instead of one with the name of the desired dataset?

Reply:
Yep!  This is wrong.

Reply:
Seems like adding a test for `update_existing_joined_hts` would be worthwhile?

Reply:
I added a test.. it was an extremely useful exercise.